onnxruntime/core/common/logging/sinks/ostream_sink.cc

```diff
@@ -31,8 +31,11 @@ void OStreamSink::SendImpl(const Timestamp& timestamp, const std::string& logger
   //
   // Going with #2 as it should scale better at the cost of creating the message in memory first
   // before sending to the stream.
-
+#ifdef _WIN32
```

Comment:
On Windows we use WOStreamSink and on Linux we use OStreamSink. See https://github.com/microsoft/onnxruntime/blob/8dba8e3e2435006a649a89fea4942f8a5f7d8bbc/onnxruntime/core/common/logging/sinks/clog_sink.h#L18

So, the "OStreamSink" class should be unused Windows. Maybe you can use ifdef to comment the whole class out.

Reply:
The FileSink might going to be a Issue because it extends OStreamSink